src/map/script_constants.hpp

```diff
@@ -7105,6 +7105,43 @@
 	export_constant(HAT_EF_SUBJECT_AURA_WHITE);
 	export_constant(HAT_EF_SUBJECT_AURA_RED);
 	export_constant(HAT_EF_C_SHINING_ANGEL_WING);
+	export_constant(HAT_EF_MAGIC_STAR_TW);
+	export_constant(HAT_EF_DIGITAL_SPACE);
+	export_constant(HAT_EF_SLEIPNIR);
+	export_constant(HAT_EF_C_MAPLE_WHICH_FALLS_RD);
+	export_constant(HAT_EF_MAGICCIRCLERAINBOW);
+	export_constant(HAT_EF_SNOWFLAKE_TIARA);
+	export_constant(HAT_EF_MIDGARTS_GLORY);
+	export_constant(HAT_EF_LEVEL99_TIGER);
+	export_constant(HAT_EF_LEVEL160_TIGER);
+	export_constant(HAT_EF_FLUFFYWING);
+	export_constant(HAT_EF_C_GHOST_EFFECT);
+	export_constant(HAT_EF_C_POPPING_PORING_AURA);
```

Comment:
export_constant(HAT_EF_ResonateTaego);
The same reason.

Reply:
done